### DISCUSSION TITLE
How to create a group list in my defineConfig?

### DISCUSSION THREAD

**Discussion by Jimerson21** [tinacms/tinacms#3851] [Help] [answered]
Hello everyone! I am new to TinaCMS and I am having problems trying to create a group list. 

I don't quite understand how it works. I would appreciate some help!

Note: I am working with Hugo.

This is my code:

![image](https://user-images.githubusercontent.com/58362749/234029820-e42bb329-d048-45d5-9b42-f93a3bea8b38.png)

**Comment by jamespohalloran** [comment, marked as answer]
[This playground](https://tina-gql-playground.vercel.app/object-list-data?markdownCode=---%0Atestimonials%3A%0A++-+author%3A+Livvy+Abby%0A++++role%3A+CEO%0A++++quote%3A+Lorem+ipsum+dolor+sit+amet+consectetur+adipisicing+elit.+Nemo+expedita+voluptas+culpa+sapiente+alias+molestiae.+Numquam+corrupti+in+laborum+sed+rerum+et+corporis.%0A---&schemaCode=import+%7B+defineSchema%2C+defineConfig+%7D+from+%27tinacms%27%0A%0Aconst+schema+%3D+defineSchema%28%7B%0A++collections%3A+%5B%7B%0A++++label%3A+%22Post%22%2C%0A++++name%3A+%22post%22%2C%0A++++path%3A+%22posts%22%2C%0A++++fields%3A+%5B%7B%0A++++++label%3A+%22Testimonials%22%2C%0A++++++name%3A+%22testimonials%22%2C%0A++++++type%3A+%22object%22%2C%0A++++++list%3A+true%2C%0A++++++ui%3A+%7B%0A++++++++%2F%2F+This+allows+the+customization+of+the+list+item+UI%0A++++++++%2F%2F+Data+can+be+accessed+by+item%3F.%3CName+of+field%3E%0A++++++++itemProps%3A+%28item%29+%3D%3E+%7B%0A++++++++++return+%7B+label%3A+%60%24%7Bitem%3F.author%7D++%28+%24%7Bitem%3F.role%7D+%29+%60%7D%0A++++++++%7D%2C%0A++++++++%2F%2F+Setting+a+default+will+auto-populate+new+items+with+the+given+values%0A++++++++defaultItem%3A+%7B%0A++++++++++author%3A+%22Judith+Black%22%2C%0A++++++++++role%3A+%22CEO%22%2C%0A++++++++++quote%3A+%22Lorem+ipsum+dolor+sit+amet+consectetur+adipisicing+elit.+Nemo+expedita+voluptas+culpa+sapiente+alias+molestiae.+Numquam+corrupti+in+laborum+sed+rerum+et+corporis.%22%0A++++++++%7D%0A++++++%7D%2C%0A++++++fields%3A+%5B%0A++++++++%7B%0A++++++++++label%3A+%22Author%22%2C%0A++++++++++name%3A+%22author%22%2C%0A++++++++++type%3A+%22string%22%0A++++++++%7D%2C%0A++++++++%7B%0A++++++++++label%3A+%22Role%22%2C%0A++++++++++name%3A+%22role%22%2C%0A++++++++++type%3A+%22string%22%0A++++++++%7D%2C%0A++++++++%7B%0A++++++++++label%3A+%22Quote%22%2C%0A++++++++++name%3A+%22quote%22%2C%0A++++++++++type%3A+%22string%22%2C%0A++++++++++ui%3A+%7B%0A++++++++++++component%3A+%22textarea%22%0A++++++++++%7D%0A++++++++%7D%0A++++++%5D%0A++++%7D%5D%0A++%7D%5D%0A%7D%29%0Aexport+default+defineConfig%28%7B+schema+%7D%29%3B%0A&reactCode=import+*+as+React+from+%27react%27%0Aimport+%7B+useTina+%7D+from+%27tinacms%2Fdist%2Freact%27%0A%0A%2F%2F+Note%3A+In+most+cases%2C+this+query+would+be%0A%2F%2F+autogenerated+with+%60client.queries.%3CCollection%3E%60%0A%2F%2F+https%3A%2F%2Ftina.io%2Fdocs%2Ffeatures%2Fdata-fetching%2F%0Aconst+QUERY+%3D+%60query+%7B%0A++post%28relativePath%3A+%22hello-world.md%22%29+%7B%0A++++testimonials+%7B%0A++++++author%0A++++++role%0A++++++quote%0A++++%7D%0A++%7D%0A%7D%60%0A%0Aexport+default+function+Page%28props%29+%7B%0A++const+%7Bdata%2C+isLoading%7D+%3D+useTina%28%7B+query%3A+QUERY%2C%0A++++variables%3A+%7B%7D%2C%0A++++data%3A+props.data%0A++%7D%29%0A%0A++if%28isLoading%29+%7B%0A++++return+%3Cdiv%3ELoading...%3C%2Fdiv%3E%0A++%7D%0A%0A++return+%28%0A++%3Cdiv+className%3D%22relative%22%3E%0A++%7Bdata.post.testimonials%3F.map%28testimonial+%3D%3E+%28%3Cblockquote+className%3D%22mt-10%22%3E%0A++++%3Cdiv+className%3D%22max-w-3xl+mx-auto+text-center+text-xl+leading-9+font-medium+text-gray-900%22%3E%0A++++++%3Cp%3E%0A++++++++%26ldquo%3B%7Btestimonial%3F.quote%7D%26rdquo%3B%0A++++++%3C%2Fp%3E%0A++++%3C%2Fdiv%3E%0A++++%3Cfooter+className%3D%22mt-8%22%3E%0A++++++%3Cdiv+className%3D%22md%3Aflex+md%3Aitems-center+md%3Ajustify-center%22%3E%0A++++++++%3Cdiv+className%3D%22md%3Aflex-shrink-0%22%3E%0A++++++++++%3Cimg%0A++++++++++++className%3D%22mx-auto+h-10+w-10+rounded-full%22%0A++++++++++++src%3D%22https%3A%2F%2Fimages.unsplash.com%2Fphoto-1494790108377-be9c29b29330%3Fixlib%3Drb-1.2.1%26ixid%3DeyJhcHBfaWQiOjEyMDd9%26auto%3Dformat%26fit%3Dfacearea%26facepad%3D2%26w%3D256%26h%3D256%26q%3D80%22%0A++++++++++++alt%3D%22%22%0A++++++++++%2F%3E%0A++++++++%3C%2Fdiv%3E%0A++++++++%3Cdiv+className%3D%22mt-3+text-center+md%3Amt-0+md%3Aml-4+md%3Aflex+md%3Aitems-center%22%3E%0A++++++++++%3Cdiv+className%3D%22text-base+font-medium+text-gray-900%22%3E%7Btestimonial%3F.author%7D%3C%2Fdiv%3E%0A%0A++++++++++%3Csvg+className%3D%22hidden+md%3Ablock+mx-1+h-5+w-5+text-indigo-600%22+fill%3D%22currentColor%22+viewBox%3D%220+0+20+20%22%3E%0A++++++++++++%3Cpath+d%3D%22M11+0h3L9+20H6l5-20z%22+%2F%3E%0A++++++++++%3C%2Fsvg%3E%0A%0A++++++++++%3Cdiv+className%3D%22text-base+font-medium+text-gray-500%22%3E%7Btestimonial%3F.role%7D%3C%2Fdiv%3E%0A++++++++%3C%2Fdiv%3E%0A++++++%3C%2Fdiv%3E%0A++++%3C%2Ffooter%3E%0A++%3C%2Fblockquote%3E%29%29%7D%0A%3C%2Fdiv%3E%0A%29%0A%7D) provides a similar example.

You will model `skill` as an object, with `list: true`

**Reply by Jimerson21** [reply]
thank you very much @jamespohalloran
